pollination_streamlit/api/client.py

```diff
@@ -1,22 +1,23 @@
 import os
 import typing as t
 from io import BytesIO
-
+from pollination_io.api.client import ApiClient as ApiClientIO
+from warnings import warn
 import requests
 
 DEFAULT_HOST = os.getenv('POLLINATION_API_URL',
                          'https://api.pollination.cloud')
 
 
-class ApiClient():
-
+class ApiClient(ApiClientIO):
+    
     def __init__(self, host: str = DEFAULT_HOST, api_token: str = None, jwt_token: str = None):
-        if host[-1] == '/':
-            host = host[:-1]
-
-        self._host = host
-        self.api_token = api_token
-        self.jwt_token = jwt_token
+        warn(f'{self.__class__.__name__} will be deprecated. Use pollination-io instead.', 
+             DeprecationWarning, 
+             stacklevel=2)
+        super().__init__(host=host,
+                         api_token=api_token,
+                         jwt_token=jwt_token)
 
     @property
```

Comment:
@AntonelloDN, you can remove all the properties and methods except for the constructor from all the classes since they have been implemented in the base class.

Reply:
Done!